### ISSUE TITLE
How do I assign an issue to myself?

### ISSUE THREAD

**Issue by qpnaosc** [reticulatedpines/rlab_issues_test#3]
I cannot see a function to do so. Can I assign myself my own issue? That is a needed function

**Comment by reticulatedpines** [comment]
![assignees](https://github.com/reticulatedpines/rlab_issues_test/assets/33672591/a2c9be2b-b5f7-434a-91eb-33138cb8d279)

Like that, in theory - if you don't see this interface I will need to change settings I guess

**Comment by qpnaosc** [comment]
nope it is not there
![Screenshot (62)](https://github.com/reticulatedpines/rlab_issues_test/assets/2410896/b483e7e0-e0f3-4a65-8d26-f14c72712cf1)

**Comment by reticulatedpines** [comment]
I've added you as a collaborator, you need to accept.  This might not immediately give you access, let me know.  I think I'll have to fiddle with collab settings for you.

**Comment by qpnaosc** [comment]
so just to clarify, is there a way that people can assign themselves to an issue without being a collaborator first? Also: can you assign an issue to multiple people?

**Comment by qpnaosc** [comment]
(the answer to that second question is yes)

**Comment by reticulatedpines** [comment]
> so just to clarify, is there a way that people can assign themselves to an issue without being a collaborator first?

I don't think so (maybe you can sidestep this with github orgs, but this isn't relevant to your question).  You also shouldn't need to assign it to yourself (or anyone) in order to comment or close it.  So assigning isn't itself that important.  However, you probably need to be listed as a collaborator to do those actions, too.

**Comment by reticulatedpines** [comment]
Counter-question: do you really want anyone on the internet to be able to close all your tickets? :)

**Comment by reticulatedpines** [comment]
Also worth noting: [Ryuk228](https://github.com/Ryuk228) was able to open an issue without being a collaborator, and can comment on at least that issue (I believe all issues).  They can probably close issues that they opened, too - but shouldn't be able to close others.

**Comment by qpnaosc** [comment]
Yeah I suppose so. Saves two people ordering the same thing but that's probably not a problem very often
